sensors/video/sensorhub-driver-v4l/src/main/java/org/sensorhub/impl/sensor/v4l/V4LCameraDriver.java

```diff
@@ -49,7 +49,7 @@ public class V4LCameraDriver extends AbstractSensorModule<V4LCameraConfig>
         }
         catch (Exception e)
         {
-            e.printStackTrace();
+            System.out.println(e.getMessage());
```

Comment:
The exception should be logged using slf4j. (Generally speaking you should not use `System.out` for anything, since logging can be set up to go to a variety of destinations.)

I recommend changing that line to something like this:

```
LoggerFactory.getLogger(V4LCameraDriver.class).error("Unable to load native v4l library", e);
```

(where LoggerFactory is org.slf4j.LoggerFactory)